### PR TITLE
Frontend web development html css js

### PR DESCRIPTION
**Pull request by darshan4903**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #issue_number_here 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Web_Development/FrontEnd/CSS Hover Effect/CSS Hover Effect .md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+#### 👉🏽HTML File
+```
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" href="style.css">
+</head>
+<body>
+<p>GWOC : <a href="https://gwoc.girlscript.tech/">girlscript.com</a></p>
+</body>
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/CSS Hover Effect/CSS Hover Effect .md:63]:
```
<body>
<p>GWOC : <a href="https://gwoc.girlscript.tech/">girlscript.com</a></p>
<br>
<p>Link: Blue</p>
<p>Visited: Green</p>
<p>Hover: Red</p>
<p>Active: Yellow</p>
</body>
```

Comment (kashishahuja2002) [reply Web_Development/FrontEnd/CSS Hover Effect/CSS Hover Effect .md:63]: Show blue in blue color, green in green and so on...

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]

**Comment by darshan4903** [comment]
updated as per suggestion @kashishahuja2002

**Review by kashishahuja2002 — APPROVED** [review]
LGTM.

**Review by 2024-SANDHYA — CHANGES_REQUESTED** [review]
The content is really very short & not structured. Start with what, why Hover effects & then the syntax, also avoid using excessive emojis, keep it a bit professional, please. The rest is all good.

**Comment by darshan4903** [comment]
Ok I will structure it and remove the emojis but what else content should I add?
Should I add more examples showing hover effect?

**Comment by 2024-SANDHYA** [comment]
> 
> 
> Ok I will structure it and remove the emojis but what else content should I add? Should I add more examples showing hover effect?

No, don't add more examples, add a bit theory, that what are hover effects, how they help and you know stuff like that not much just 5-6 lines will work.

File: Web_Development/FrontEnd/CSS Hover Effect/CSS Hover Effect .md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="box">
+         <h1>Hello World</h1>
+    </div>
+</body>
+</html>
+```
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/CSS Hover Effect/CSS Hover Effect .md:31]: Code blocks should have syntax.

File: Web_Development/FrontEnd/CSS Hover Effect/CSS Hover Effect .md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+}
+h1{
+    color: aqua;
+    text-align: center;
+    margin: auto;
+}
+h1:hover{
+    color: brown;
+}
+```
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/CSS Hover Effect/CSS Hover Effect .md:48]: Code blocks should have syntax.

File: Web_Development/FrontEnd/CSS Hover Effect/CSS Hover Effect .md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+<link rel="stylesheet" href="style.css">
+</head>
+<body>
+<p style="color: white;">GWOC : <a href="https://gwoc.girlscript.tech/">girlscript.com</a></p>
+<p class="visited">Visited: Green</p>
+<p class="hover">Hover: Red</p>
+<p class="active">Active: Yellow</p>
+</body>
+</html>
+```
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/CSS Hover Effect/CSS Hover Effect .md:68]: Code blocks should have syntax.

File: Web_Development/FrontEnd/CSS Hover Effect/CSS Hover Effect .md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+  .active{
+    color: yellow;
+  }
+  .visited{
+    color: green;
+  }
+  body{
+    background-color: black;
+  }
+```
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/CSS Hover Effect/CSS Hover Effect .md:92]: Code blocks should have syntax.

**Review by code-reaper08 — CHANGES_REQUESTED** [review]
Need a bit more explanation on the topic.
Other things LGTM ✨

**Comment by darshan4903** [comment]
Updated @2024-SANDHYA @code-reaper08

**Review by 2024-SANDHYA — CHANGES_REQUESTED** [review]
Reference links missing.

**Comment by darshan4903** [comment]
updated @2024-SANDHYA

**Review by 2024-SANDHYA — APPROVED** [review]

**Review by code-reaper08 — APPROVED** [review]

**Pull request merged**